src/components/Copyright/Copyright.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import LayerService from 'react-spatial/LayerService';
+
+const propTypes = {
+  layerService: PropTypes.instanceOf(LayerService).isRequired,
+};
+
+class Copyright extends Component {
+  constructor(props) {
+    super(props);
+    const { layerService } = this.props;
+    this.layerService = layerService;
```

Comment:
You shouldn't set a class property from a props. You will loose the reference if the props change.